app/controllers/publishers/uphold_controller.rb

```diff
@@ -85,6 +85,13 @@ def create
       redirect_to(home_publishers_path, alert: t(".uphold_error", message: e.message))
     end
 
+    def update
+      send_emails = params[:uphold_connection_send_emails]
+      # If the value is blank then user has opted out of emails.
```

Comment:
IMO you should make it 100 years from now (or some other unreasonable date in the future). If the value is blank, then that means the user hasn't been sent emails